dendrite-config.yaml

```diff
@@ -85,6 +85,14 @@ global:
     # Whether outbound presence events are allowed, e.g. sending presence events to other servers
     enable_outbound: false
 
+  # Whether this instance sends anonymous usage stats
+  report_stats: false
+
+  # The endpoint to report the anonymized homeserver usage statistics to.
+  # Defaults to https://matrix.org/report-usage-stats/push
+  #
+  report_stats_endpoint: https://matrix.org/report-usage-stats/push
```

Comment:
Might be neater to have it be:
```yaml
report_stats:
    enabled: true
    endpoint: ...
```

This matches the grouping on other things like `server_notices`.